thunder/benchmarks/targets.py

```diff
@@ -812,3 +815,37 @@ def test_resnet50(benchmark, executor: Callable, compute_type: ComputeType):
     fn = executor(b.fn())
 
     benchmark_for_compute_type(compute_type, benchmark, fn, args, kwargs)
+
+
+#
+# Torchbench benchmarks
+#
+
+torchbench_models = []
+if importlib.util.find_spec("torchbenchmark"):
+    import torchbenchmark
+
+    torchbench_models = dir(torchbenchmark.models)
```

Comment:
I wonder if there is another way to fetch models as `dir` also pulls in following details from `torchbenchmark.models` - `'__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '_list_models_without_import', '_model_name'`.
```python
>>> import torchbenchmark
>>> dir(torchbenchmark.models)
['BERT_pytorch', 'Background_Matting', 'LazyImport', 'LearningToPaint', 'Path', 'Super_SloMo', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', '_list_models_without_import', '_model_name', 'alexnet', 'basic_gnn_edgecnn', 'basic_gnn_gcn', 'basic_gnn_gin', 'basic_gnn_sage', 'cm3leon_generate', 'dcgan', 'demucs', 'densenet121', 'detectron2_fasterrcnn_r_101_c4', 'detectron2_fasterrcnn_r_101_dc5', 'detectron2_fasterrcnn_r_101_fpn', 'detectron2_fasterrcnn_r_50_c4', 'detectron2_fasterrcnn_r_50_dc5', 'detectron2_fasterrcnn_r_50_fpn', 'detectron2_fcos_r_50_fpn', 'detectron2_maskrcnn', 'detectron2_maskrcnn_r_101_c4', 'detectron2_maskrcnn_r_101_fpn', 'detectron2_maskrcnn_r_50_c4', 'detectron2_maskrcnn_r_50_fpn', 'dlrm', 'doctr_det_predictor', 'doctr_reco_predictor', 'drq', 'fastNLP_Bert', 'functorch_dp_cifar10', 'functorch_maml_omniglot', 'hf_Albert', 'hf_Bart', 'hf_Bert', 'hf_Bert_large', 'hf_BigBird', 'hf_DistilBert', 'hf_GPT2', 'hf_GPT2_large', 'hf_Longformer', 'hf_Reformer', 'hf_Roberta_base', 'hf_T5', 'hf_T5_base', 'hf_T5_generate', 'hf_T5_large', 'hf_Whisper', 'hf_clip', 'hf_distil_whisper', 'importlib', 'lennard_jones', 'llama', 'llama_v2_7b_16h', 'llava', 'maml', 'maml_omniglot', 'microbench_unbacked_tolist_sum', 'mnasnet1_0', 'mobilenet_v2', 'mobilenet_v2_quantized_qat', 'mobilenet_v3_large', 'moco', 'moondream', 'nanogpt', 'nvidia_deeprecommender', 'opacus_cifar10', 'os', 'phlippe_densenet', 'phlippe_resnet', 'pyhpc_equation_of_state', 'pyhpc_isoneutral_mixing', 'pyhpc_turbulent_kinetic_energy', 'pytorch_CycleGAN_and_pix2pix', 'pytorch_stargan', 'pytorch_unet', 'resnet152', 'resnet18', 'resnet50', 'resnet50_quantized_qat', 'resnext50_32x4d', 'sam', 'sam_fast', 'shufflenet_v2_x1_0', 'simple_gpt', 'simple_gpt_tp_manual', 'soft_actor_critic', 'speech_transformer', 'squeezenet1_1', 'stable_diffusion_text_encoder', 'stable_diffusion_unet', 'sys', 'tacotron2', 'timm_efficientdet', 'timm_efficientnet', 'timm_nfnet', 'timm_regnet', 'timm_resnest', 'timm_vision_transformer', 'timm_vision_transformer_large', 'timm_vovnet', 'torch_multimodal_clip', 'tts_angular', 'vgg16', 'vision_maskrcnn', 'yolov3']
```

Reply:
Oh actually I just found out that there is a way by using an hidden fn of torchbench, let me update the import